sdk/keyvault/Azure.Security.KeyVault.Secrets/aot/Azure.Security.KeyVault.Secrets.Aot.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PublishAot>true</PublishAot>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+    <IsClientLibrary>false</IsClientLibrary>
+    <IsTestSupportProject>true</IsTestSupportProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TrimmerRootAssembly Include="Azure.Security.KeyVault.Secrets" />
```

Comment:
I am assuming this is how we explain what library should be trimmed?

Reply:
I assume so. It's basically what @m-redding told me to do.

Reply:
Usually, the trimmer only looks at the code paths used by application code, but this just tells the trimmer to look at the whole assembly instead.

Reply:
It's used primarily for this purpose - testing the trim-compatibility of a library. An actual application wanting to use Keyvault in an AOT app wouldn't want to do this, since they only care about the code paths they're using being trim-compatible.